### DISCUSSION TITLE
Can LNP Node be monetised same way that regular LN node can?

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/presentations#33] [Q&A]
### TL;DR
Summary on the WIP on monetization
- routing fees as in any other nodes
- watchtower fees as in other nodes

Unique features:
- fees for running public infrastructure of RGB protocol (Bifrost)
- DEX operator node fees
- liquidity pool provider fees
- in longer term - fees for storm-related features (trustless storage)

### TS;WM 
There is nothing specific inside any LN node that allows their monetisation, it’s more about the Lightning Network itself and protocols where you have things like routing fee or where you can build services on top of the node which you can monetise with LN payments.
In this regard, the LNP Node is not different from any other nodes, so it can be monetised in the same way, as monetisation is not node-specific, but protocol-specific and specific to the end-user services that are implemented on top of the node by companies or users.  

However, outside of LNP Node, the LNP/BP Standards Association has been building a set of protocols that would add a lot of monetisation options to the LN itself:
1. First protocol is **RGB**, which unlocks the functionality of DEX inside the Lightning Network, where LNP Node will allow you to act as a part of a DEX, with further extending the possibilities to create liquidity pools for different RGB assets. 

2. Next protocol is called **Storm** and is made (not fully implemented yet, but we already have required faucets ready) for storage and messaging, meaning that LNP Node can be a part of this storage provider and we will have an infrastructure around the node, which will allow different additional monetisation options not present in LN now.
